tests/testimonialCarousel.test.ts

```diff
@@ -14,21 +14,21 @@ describe('Testimonial - Carousel', () => {
 
         await testimonialActive.dragAndDrop({ x: 100, y: 0 })
         const coryDrag = await $('.swiper-slide-next .slick-slide').$('.elementskit-author-name=Cory Clemmons')
+        
         await browser.pause(2000)       
         await testimonialActive.dragAndDrop({ x: -100, y: 0 })
         await browser.pause(3000)
-
         await expect(coryDrag).toBeExisting()
         await expect(eric).toBeExisting()
 
         // Sliding with buttons
 
         await $('.slick-next').click()
```

Comment:
@JuliaCabrera23 please add an await browser.pause(3000) after this line due if is not, the test failed

[chrome 109.0.5414.75 windows #0-0] 1) Testimonial - Carousel Check the testimonial carousel
[chrome 109.0.5414.75 windows #0-0] element not interactable
  (Session info: chrome=109.0.5414.75)